README.md

```diff
@@ -62,5 +62,5 @@ Make sure jQuery is loaded ahead of bootstrap.js.
 
 
 ## External Resources
-
+- [Orbit](https://github.intern.weebly.net/weebly/orbit) - Orbit is being imported with the css file `css/orbit.css`. Do not make changes directly to this file.
```

Comment:
Not req for this PR. But at the end of this project, before we hand off, we should document the steps required to roll in the latest version of Orbit's CSS. I've created a JIRA task for this. This will make the handoff cleaner and give the ability for other teams to maintain the site w/o significant DE hand-holding.
https://jira.weebly.net/browse/DE-1083